app/controllers/owners/restaurants_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Owners::RestaurantsController < ApplicationController
+
+  def index
+    @owner = current_user.id
```

Comment:
okay since you need `@owner` in both actions you can do the following

1. Create a private method
```ruby
def owner
  @owner = User.find(current_user.id)
end
```

2. Add a `before_action`

```ruby
before_action :owner
```

What that means is that you can get rid of line 4 and line 10 in this controller.

I'm guessing you're using `@owner_id` in the view, after you make the above change, go to the view and use `@owner.id`